src/AboutPage/About.tsx

```diff
@@ -2,7 +2,46 @@ import React from 'react'
 
 function About() {
   return (
-    <div>About</div>
+    <body>
+      <section className='flex justify-center items-center h-screen w-screen flex-col'>
+        <h1 className=' text-5xl m-10'>About</h1>
+        <img src="/src/images/hero-img.png" className='w-5/6 h 3/6'></img>
+      </section>
+
+      <section className='flex flex-col justify-center items-center'>
+        <article className='flex flex-row w-5/6 m-20'>
+          <div className='flex flex-col align-left '>
+            <h2 className='text-sm'>About us</h2>
+            <h3 className='text-3xl'>Our Restaurant</h3>
```

Comment:
Improper use of headings. For accessibility purposes, headings are meant to define subsections of content. "Our Restaurant" is not a subsection of "About Us". There are different ways of fixing this—because "About Us" should be associated with the "Our Restaurant" section, you can put it in a span within the h2, so something like `<h2><span>About Us</span> Our Restaurant</h2>` and then use Tailwind classes to style the layout.